src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <li key={movie.title}>
```

Comment:
Use id for it
![image](https://user-images.githubusercontent.com/91411692/231416061-c6323648-a354-4b9c-8c50-6313108d806a.png)
```suggestion
      <li key={movie.imdbId}>
```